Source/Core/Common/Assembler/GekkoLexer.h

```diff
@@ -0,0 +1,185 @@
+#pragma once
+
+#include <algorithm>
+#include <array>
+#include <deque>
+#include <optional>
+#include <string_view>
+#include <type_traits>
+#include <vector>
+
+#include "Common/Assembler/AssemblerShared.h"
+#include "Common/Assembler/AssemblerTables.h"
+#include "Common/CommonTypes.h"
+
+namespace Common::GekkoAssembler::detail
+{
+void ConvertStringLiteral(std::string_view literal, std::vector<u8>& out_vec);
```

Comment:
The out parameter should be passed as pointer based on [Dolphin coding style](https://github.com/dolphin-emu/dolphin/blob/master/Contributing.md#cpp-code-functions).